frontend/src/components/browse/BrowseContent.js

```diff
@@ -113,6 +126,46 @@ export default function BrowseContent({
     }
   }, []);
 
+  const router = useRouter();
+  const hasQueryParams = !!router.query;
+
+  /**
+   * Support the functionality of a user entering
+   * a provided URL, that already has URL encoded
+   * query params from a filter in it. In this use
+   * case, we should automatically set the filters dynamically. Ensure
+   * that this isn't invoked on extraneous renders.
+   */
+  let hasFilteredFromQueryParams = false;
+  useEffect(() => {
+    if (hasQueryParams && !hasFilteredFromQueryParams) {
+      // Update the state of the visual filters, like Select, Dialog, etc
+      // Then actually fetch the data. We need a way to map what's
+      // in the query param, to what's UI control is present on the screen. For
+      // example, if we have a MultiLevelSelect dialog representing categories
+      // and we have a ?&category=Food waste, then we need to update the
+      // the MultiLevelSelect dialog's selection to that value somehow.
+
+      // For each query param option, ensure that it's
+      // split into array before spreading onto the new filters object
+      const queryObject = _.cloneDeep(router.query);
+      for (const [key, value] of Object.entries(queryObject)) {
+        if (value.indexOf(",") > 0) {
+          queryObject[key] = value.split(",");
+        }
+      }
+
+      let newFilters = { ...queryObject };
+
+      // TODO(piper): fix hardcoded projects here...
+      // Apply new filters with the query object immediately:
+      handleApplyNewFilters("projects", newFilters, false, state.urlEnding["projects"]);
```

Comment:
@piperchester -- need to remove the hardcoded "projects" bit here: 
 
```suggestion
      // Apply new filters with the query object immediately:
      handleApplyNewFilters("projects", newFilters, false, state.urlEnding["projects"]);
```

Reply:
@piperchester -- need to remove the hardcoded "projects" bit here: